### PR TITLE
fix: clarify cross_model comment and add git-workflow skill

### PR DESCRIPTION
**Pull request by fredericboyer**
## Summary
- Updated `cross_model` comment in all 6 reviewer agent definitions (knuth, szabo, brooks in both `templates/agents/` and `.dev-team/agents/`) to say "recommended true" for clarity on intended future value
- Added `.claude/skills/git-workflow/SKILL.md` — project-specific skill documenting worktree isolation for safe parallel team work (prevents branch contamination when multiple agents share the same repo)

## Test plan
- [x] `npm test` passes (308/308)
- [x] YAML frontmatter parses correctly
- [x] Skill file follows SKILL.md format

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: templates/agents/dev-team-knuth.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ name: dev-team-knuth
 description: Quality auditor. Use to audit code for correctness gaps, missing test coverage, boundary conditions, and unproven assumptions. Read-only — identifies what is missing or unproven, does not write code or tests.
 tools: Read, Grep, Glob, Bash, Agent
 model: opus
-cross_model: false # recommended when multi-model runtime available (ADR-023)
+cross_model: false # recommended true when multi-model runtime available (ADR-023)
```
[the remark this answers is not in the document]
Comment (fredericboyer) [reply templates/agents/dev-team-knuth.md]: Acknowledged. The `cross_model` field will be removed entirely from agent definitions in the v1.0.0 release PR.

File: .dev-team/agents/dev-team-szabo.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ name: dev-team-szabo
 description: Security auditor. Use to review code for vulnerabilities, audit auth flows, analyze attack surfaces, and assess dependency risks. Read-only — does not modify code.
 tools: Read, Grep, Glob, Bash, Agent
 model: opus
-cross_model: false # recommended when multi-model runtime available (ADR-023)
+cross_model: false # recommended true when multi-model runtime available (ADR-023)
```
[the remark this answers is not in the document]
Comment (fredericboyer) [reply .dev-team/agents/dev-team-szabo.md]: Acknowledged. The `cross_model` field will be removed entirely from agent definitions in the v1.0.0 release PR.

File: .dev-team/agents/dev-team-knuth.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ name: dev-team-knuth
 description: Quality auditor. Use to audit code for correctness gaps, missing test coverage, boundary conditions, and unproven assumptions. Read-only — identifies what is missing or unproven, does not write code or tests.
 tools: Read, Grep, Glob, Bash, Agent
 model: opus
-cross_model: false # recommended when multi-model runtime available (ADR-023)
+cross_model: false # recommended true when multi-model runtime available (ADR-023)
```
[the remark this answers is not in the document]
Comment (fredericboyer) [reply .dev-team/agents/dev-team-knuth.md]: Acknowledged. The `cross_model` field will be removed entirely from agent definitions in the v1.0.0 release PR.

File: .claude/skills/git-workflow/SKILL.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+## Workflow
+
+1. **Create worktree** from latest `origin/main`
+2. **Make changes** — edit, test, commit inside the worktree
+3. **Push** with `-u origin <branch-name>`
+4. **Create PR** with `Closes #<issue>` in body
+5. **Clean up** after push:
+   ```bash
+   cd /Users/fredericboyer/Projects/dev-team
```
[the remark this answers is not in the document]
Comment (fredericboyer) [reply .claude/skills/git-workflow/SKILL.md]: Acknowledged. The `cross_model` field will be removed entirely from agent definitions in the v1.0.0 release PR.

File: templates/agents/dev-team-brooks.md [hunk 5]
```diff
@@ -3,7 +3,7 @@ name: dev-team-brooks
 description: Architect and quality attribute reviewer. Use to review architectural decisions, challenge coupling and dependency direction, validate changes against ADRs, and assess quality attributes (performance, maintainability, scalability). Always-on for all non-test code changes. Read-only — does not modify code.
 tools: Read, Grep, Glob, Bash, Agent
 model: opus
-cross_model: false # recommended when multi-model runtime available (ADR-023)
+cross_model: false # recommended true when multi-model runtime available (ADR-023)
```
[the remark this answers is not in the document]
Comment (fredericboyer) [reply templates/agents/dev-team-brooks.md]: Fixed in release PR #247: Removed `cross_model` field entirely from all agent defs. Field will only be added when set to a valid model identifier string per ADR-023.

File: .claude/skills/git-workflow/SKILL.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
+---
+name: git-workflow
+description: Safe parallel git workflow — worktree isolation, branch naming, and cleanup for team work.
+user_invocable: false
```
[the remark this answers is not in the document]
Comment (fredericboyer) [reply .claude/skills/git-workflow/SKILL.md]: Fixed in release PR #247: Changed `user_invocable: false` to `user_invocable: true` to match all other skills in the repo.
